components/Portfolio/Trading/Overview/index.tsx

```diff
@@ -164,31 +165,20 @@ const PortfolioDropdown: React.FC<PDProps> = styled(({ className, setOptions, op
     }
 `;
 
-const Overview: FC<{
-    selectedTracer?: Tracer | undefined;
-}> = ({ selectedTracer }) => {
+const Overview: FC = () => {
     const { account } = useWeb3();
     const [currentPortfolio, setCurrentPortfolio] = useState(1);
     const [currentPNL, setCurrentPNL] = useState(1);
     const { factoryState: { tracers } = initialFactoryState } = useContext(FactoryContext);
-    const [tracersAddress, setTracersAddress] = useState<string[]>([]);
-    const balances = selectedTracer?.getBalance() ?? defaults.balances;
-    const fairPrice = selectedTracer?.getFairPrice() ?? defaults.fairPrice;
-    const {
-        // omeState,
-        // omeDispatch = () => {
-        //     console.error('OME dispatch is undefined');
-        // },
-        filledOrders,
-    } = useContext(OMEContext);
+    const [fetchedTracers, setFetchedTracers] = useState<any>([]);
 
     const fetchTracers = setTimeout(async function getInfo() {
-        const temp: string[] = [];
+        const tempTracers = [];
         for (const key of Object.keys(tracers)) {
-            temp.push(tracers[key].address);
+            tempTracers.push(tracers[key]);
         }
-        setTracersAddress(temp);
-    }, 100);
+        setFetchedTracers(tempTracers);
+    }, 1000);
```

Comment:
This is a pretty big hack but it is ok since I know making sure the state is updated and things correctly change is hard. Will have to be updated in the near future. But to allow you to continue development it is ok.